From e89e7655a197d28df49da2be7e2003436cf52197 Mon Sep 17 00:00:00 2001
From: Ignacio Larrain <ilarrain@gmail.com>
Date: Tue, 22 Aug 2023 11:11:56 -0400
Subject: [PATCH] drm/panel: Fix default values for Waveshare 7.9 inch DSI
 touchscreen (#5565)

This fixes touchscreen calibration, axis swapping and inversion.

As referenced in https://github.com/raspberrypi/linux/issues/5550
---
 .../dts/overlays/vc4-kms-dsi-waveshare-panel-overlay.dts    | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/arch/arm/boot/dts/overlays/vc4-kms-dsi-waveshare-panel-overlay.dts
+++ b/arch/arm/boot/dts/overlays/vc4-kms-dsi-waveshare-panel-overlay.dts
@@ -93,10 +93,10 @@
 				   <&touch>, "touchscreen-size-x:0=800",
 				   <&touch>, "touchscreen-size-y:0=480";
 		7_9_inch = <&panel>, "compatible=waveshare,7.9inch-panel",
-				   <&touch>, "touchscreen-size-x:0=400",
-				   <&touch>, "touchscreen-size-y:0=1280",
+				   <&touch>, "touchscreen-size-x:0=4096",
+				   <&touch>, "touchscreen-size-y:0=4096",
 				   <&touch>, "touchscreen-inverted-x?",
-				   <&touch>, "touchscreen-inverted-y?";
+				   <&touch>, "touchscreen-swapped-x-y?";
 		8_0_inch = <&panel>, "compatible=waveshare,8.0inch-panel",
 				   <&touch>, "touchscreen-size-x:0=800",
 				   <&touch>, "touchscreen-size-y:0=1280",
